src/video/windows/SDL_windowsvideo.c

```diff
@@ -62,6 +62,15 @@ static void SDLCALL UpdateWindowsRawKeyboard(void *userdata, const char *name, c
     WIN_SetRawKeyboardEnabled(_this, enabled);
 }
 
+static void SDLCALL UpdateWindowsRawKeyboardNoHotkeys(void *userdata, const char *name, const char *oldValue, const char *newValue)
+{
+    SDL_VideoDevice *_this = (SDL_VideoDevice *)userdata;
+    SDL_VideoData *data = _this->internal;
+    data->raw_keyboard_flag_nohotkeys = SDL_GetStringBoolean(newValue, false);
+    WIN_SetRawKeyboardEnabled(_this, data->raw_keyboard_enabled);
```

Comment:
I'm personally not too happy with this calling into the existing logic like this. The existing `internal` `raw_keyboard_*` flags are all set in `SDL_windowsrawinput.c`, I think it's probably best to follow existing patterns and add `WIN_SetRawKeyboardNoHotkeys()`, it'll lead to less confusion in the future.

And when you add `WIN_SetRawKeyboardNoHotkeys()`, you'll notice a call to it is missing from `WIN_VideoQuit()`.